dev/tools/gen_defaults/data/color_dark.json

```diff
@@ -0,0 +1,31 @@
+{
+  "version": "v0_76",
+
+  "md.sys.color.background": "md.ref.palette.neutral10",
+  "md.sys.color.error": "md.ref.palette.error80",
+  "md.sys.color.error-container": "md.ref.palette.error30",
+  "md.sys.color.inverse-on-surface": "md.ref.palette.neutral20",
+  "md.sys.color.inverse-primary": "md.ref.palette.primary40",
+  "md.sys.color.inverse-surface": "md.ref.palette.neutral90",
+  "md.sys.color.on-background": "md.ref.palette.neutral90",
+  "md.sys.color.on-error": "md.ref.palette.error20",
+  "md.sys.color.on-error-container": "md.ref.palette.error80",
+  "md.sys.color.on-primary": "md.ref.palette.primary20",
+  "md.sys.color.on-primary-container": "md.ref.palette.primary90",
+  "md.sys.color.on-secondary": "md.ref.palette.secondary20",
+  "md.sys.color.on-secondary-container": "md.ref.palette.secondary90",
+  "md.sys.color.on-surface": "md.ref.palette.neutral90",
+  "md.sys.color.on-surface-variant": "md.ref.palette.neutral-variant80",
+  "md.sys.color.on-tertiary": "md.ref.palette.tertiary20",
+  "md.sys.color.on-tertiary-container": "md.ref.palette.tertiary90",
+  "md.sys.color.outline": "md.ref.palette.neutral-variant60",
+  "md.sys.color.primary": "md.ref.palette.primary80",
+  "md.sys.color.primary-container": "md.ref.palette.primary30",
+  "md.sys.color.secondary": "md.ref.palette.secondary80",
+  "md.sys.color.secondary-container": "md.ref.palette.secondary30",
+  "md.sys.color.shadow": "md.ref.palette.neutral0",
+  "md.sys.color.surface": "md.ref.palette.neutral10",
+  "md.sys.color.surface-variant": "md.ref.palette.neutral-variant30",
+  "md.sys.color.tertiary": "md.ref.palette.tertiary80",
+  "md.sys.color.tertiary-container": "md.ref.palette.tertiary30"
+}
```

Comment:
Nit: Newline at end of all these json files?

Reply:
Caused by the token extractor script. I'll see what I can do next time I am updating the JSON files.

Reply:
Fixed.